src/Policies.cpp

```diff
@@ -336,7 +357,16 @@ void PolicyManager::remove_policy(const std::string &name)
         _registry->handler_manager()->module_remove(name);
     }
 
-    if (!policy->input_stream()->policies_count()) {
+    if (policy->input_stream()->policies_count() == 1) {
```

Comment:
ok i think this is ok, but let's comment the strategy here, i.e. "if there is only one policy left on the input stream, and that policy is the input resources policy, then remove it"

Reply:
ok